src/graphics/EInkDisplay2.cpp

```diff
@@ -256,21 +307,8 @@ bool EInkDisplay::connect()
     adafruitDisplay->init(115200, true, 40, false, SPI, SPISettings(4000000, MSBFIRST, SPI_MODE0));
     adafruitDisplay->setRotation(0);
     adafruitDisplay->setPartialWindow(0, 0, EPD_WIDTH, EPD_HEIGHT);
-#elif defined(my)
```

Comment:
Can someone please confirm that these deletions are appropriate?

Reply:
I used them for the diy e-ink targets